Marlin/Configuration.h

```diff
@@ -646,6 +646,8 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
   #define EEPROM_CHITCHAT // Please keep turned on if you can.
 #endif
 
+//#define HOST_KEEPALIVE_FEATURE // Send messages to the host when busy with long operations
```

Comment:
OctoPrint automatically supports this since OctoPrint interprets *any* output from the printer as "oh, look, it's still alive!" (and has been doing so for quite some time now). So another :+1: for having it enabled by default, it will *instantly* solve a lot of issues OctoPrint users are currently experiencing with long running commands.